extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -26,24 +26,24 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 	private _extensionContext: vscode.ExtensionContext;
 	private prompter: IPrompter;
 	private _initializeDeferred: Deferred<void> = new Deferred<void>();
-	private _apiWrapper: ApiWrapper;
 	private _openAsUntitled: boolean;
 	private _trustResourceCache: Record<string, boolean> = {};
 	private _bookTrustManager: IBookTrustManager;
+	private _apiWrapper: ApiWrapper;
 
 	public viewId: string;
 	public books: BookModel[];
 	public currentBook: BookModel;
 
-	constructor(workspaceFolders: vscode.WorkspaceFolder[], extensionContext: vscode.ExtensionContext, openAsUntitled: boolean, view: string) {
+	constructor(workspaceFolders: vscode.WorkspaceFolder[], extensionContext: vscode.ExtensionContext, openAsUntitled: boolean, view: string, apiWrapper?: ApiWrapper) {
```

Comment:
Any reason you have this be optional? The apiWrapper should never be undefined - we can guarantee that